src/components/common.js

```diff
@@ -260,43 +372,80 @@ export const comingSoon = span({
   }
 }, ['coming soon'])
 
-const commonSelectProps = {
-  theme: base => _.merge(base, {
-    colors: {
-      primary: colors.accent(),
-      neutral20: colors.dark(0.55),
-      neutral30: colors.dark(0.55)
-    },
-    spacing: { controlHeight: 36 }
-  }),
-  styles: {
-    control: (base, { isDisabled }) => _.merge(base, {
-      backgroundColor: isDisabled ? colors.dark(0.25) : 'white',
-      boxShadow: 'none'
-    }),
-    singleValue: base => ({ ...base, color: colors.dark() }),
-    option: (base, { isSelected, isFocused, isDisabled }) => _.merge(base, {
-      fontWeight: isSelected ? 600 : undefined,
-      backgroundColor: isFocused ? colors.dark(0.15) : 'white',
-      color: isDisabled ? undefined : colors.dark(),
-      ':active': { backgroundColor: colors.accent(isSelected ? 0.55 : 0.4) }
+const commonSelectProps = (menuId, isOpen) => {
+  return {
```

Comment:
I'd tell you to use the shorthand for returning a literal object here, `() => ({})`